kernel/src/mm/phys/mod.rs

```diff
@@ -64,6 +65,37 @@ static mut FRAME_ALLOCATOR_STORAGE: [u8; config::kernel::MEMORY_SIZE
 // Standalone Functions
 //==================================================================================================
 
+fn parse_physical_memory_regions(
```

Comment:
Could you add documentation to this function? to this function?

Reply:
Sure.